### ISSUE TITLE
No sudo required?

### ISSUE THREAD

**Issue by flatsiedatsie** [frawau/aioarping#1]
Am I correct in understanding this library won't need any root-like privileges to work?

**Comment by frawau** [comment]
You are not correct.  This library uses raw socket. Those usually require permission.
